class Excepcion:
    def __init__(self) -> None:
        pass